docs/plugin-vidarr.md

```diff
@@ -18,3 +18,22 @@ should be submitted to Vidarr. If null, worklow runs will always be submitted.
 If a number, in seconds, is given, then the action must have been
 (re-)generated by an olive in that window to be submitted to Vidarr. If that
 window is exceeded, they will be run as dry-run.
+
+The vidarr plugin provides an imput format: `vidarr_provenance` which 
+provides the analyis provenance data.
+To use, add to the configuration file:
```

Comment:
'add to the configuration file' suggests adding this block as secondary to the block defined at line 8. Could these two sections be combined?